app/models/article.rb

```diff
@@ -0,0 +1,8 @@
+class Article < ApplicationRecord
+  include Visible
+
+  has_many :comments, dependent: :destroy
+
+  validates :title, presence: true
+  validates :body, presence: true, length: { minimum: 10 }
+end
```

Comment:
Remember always to put an empty line at the end of the files.

Reply:
Got it :smile: